api/v1alpha1/sleepinfo_webhook_test.go

```diff
@@ -30,7 +30,7 @@ func TestValidateSleepInfo(t *testing.T) {
 	}{
 		{
 			name:          "fails - without weekdays",
-			expectedError: "empty weekdays from SleepInfo configuration",
+			expectedError: "empty weekdays and weekdaysleep or weekdaywakeup from SleepInfo configuration",
```

Comment:
We could also add the validation at webhook time, so that it will reject the creation of invalid weekday sleep or wake up. What do you think about?